src/llama-vocab.cpp

```diff
@@ -17,6 +17,12 @@
 #include <set>
 #include <unordered_map>
 
+// disable C++11 deprecation warning non-constant-expression cannot be narrowed
+#if defined(__clang__)
+#    pragma clang diagnostic push
+#    pragma clang diagnostic ignored "-Wc++11-narrowing"
+#endif
+
```

Comment:
What the warnings that are ignored by this?

Reply:
The strangest thing was that there was an error initially, but now it's gone.